core/src/test/scala/org/apache/spark/scheduler/BlacklistTrackerSuite.scala

```diff
@@ -88,6 +88,86 @@ class BlacklistTrackerSuite extends SparkFunSuite with BeforeAndAfterEach with M
     new TaskSetBlacklist(conf, stageId, clock)
   }
 
+  def configureBlacklistAndScheduler(confs: (String, String)*): Unit = {
```

Comment:
since you're never passing in `confs`, couldn't you get rid of this method, and just change the initialization in `beforeEach` to

```
listenerBusMock = mock[LiveListenerBus]
blacklist = new BlacklistTracker(listenerBusMock, conf, clock)
```